src/containers/lock/containers/LockForm/index.js

```diff
@@ -301,6 +300,18 @@ class LockContainer extends React.Component {
                                     </strong>
                                     {"."}
                                 </p>
+                                <p
+                                    style={{
+                                        color: theme.colors.darkRed,
+                                        fontWeight: "bold",
+                                        fontSize: 14,
+                                        margin: "20px auto 20px auto",
+                                        textAlign: "center",
+                                        lineHeight: "120%"
+                                    }}
+                                >
+                                    Note: amount cannot be unlocked before <br /> {unlockBy}.
```

Comment:
maybe

> Note: amount cannot be unlocked until ...

Also, do we want it in red for sure?

Reply:
I think so. It's surprising behavior compared to general user expectations.